actionpack/test/controller/caching_test.rb

```diff
@@ -344,31 +344,8 @@ def self.output_buffer=; end
 
     cache_helper.stub :controller, controller do
       cache_helper.stub :output_buffer, output_buffer do
-        assert_called_with cache_helper, :output_buffer=, [output_buffer.class.new(output_buffer)] do
-          assert_nothing_raised do
-            cache_helper.send :fragment_for, "Test fragment name", "Test fragment", &Proc.new { nil }
-          end
-        end
-      end
-    end
-  end
-
-  def test_safe_buffer
-    output_buffer = ActiveSupport::SafeBuffer.new
```

Comment:
I don't see any legit reason for using `SafeBuffer` as `@output_buffer` and its holding us back.